src/streaming/models/VideoModel.js

```diff
@@ -104,7 +104,14 @@ function VideoModel() {
                 removeEventListener(setCurrentTimeReadyStateFunction.event, setCurrentTimeReadyStateFunction.func);
             }
             _currentTime = currentTime;
-            setCurrentTimeReadyStateFunction = waitForReadyState(Constants.VIDEO_ELEMENT_READY_STATES.HAVE_METADATA, () => {
+
+            let elementReadyStateEvent = Constants.VIDEO_ELEMENT_READY_STATES.HAVE_METADATA;
```

Comment:
You'd save an assignment by making this a ternary. I'm not gonna block on that insignificant a change tho